packages/twenty-front/src/modules/page-layout/constants/DefaultWorkflowPageLayout.ts

```diff
@@ -0,0 +1,53 @@
+import { DEFAULT_WORKFLOW_PAGE_LAYOUT_ID } from '@/page-layout/constants/DefaultWorkflowPageLayoutId';
+import { type PageLayout } from '@/page-layout/types/PageLayout';
+import { PageLayoutType, WidgetType } from '~/generated/graphql';
+
+/**
+ * Default Workflow PageLayout.
```

Comment:
Docblock-style comments aren't really used in the codebase. I don't mind them personally but maybe we should remove them for consistency